front/app/containers/Admin/reporting/components/ReportBuilder/Analysis/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useState } from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { IOption } from 'typings';
+
+import usePhase from 'api/phases/usePhase';
+
+import { useReportContext } from 'containers/Admin/reporting/context/ReportContext';
+
+import PhaseFilter from 'components/UI/PhaseFilter';
+
+import { useIntl } from 'utils/cl-intl';
+
+import ProjectFilter from '../Widgets/_shared/ProjectFilter';
+import QuestionSelect from '../Widgets/_shared/QuestionSelect';
+import widgetMessages from '../Widgets/messages';
+
+import Analyses from './Analyses';
+import messages from './messages';
+
+const Analysis = ({ selectedLocale }: { selectedLocale: string }) => {
+  const { projectId: contextProjectId, phaseId: contextPhaseId } =
+    useReportContext();
+
+  const [projectId, setProjectId] = useState<string | undefined>(
+    contextProjectId
+  );
+  const [phaseId, setPhaseId] = useState<string | undefined>(contextPhaseId);
+  const [questionId, setQuestionId] = useState<string | undefined>(undefined);
+
+  const { data: phase } = usePhase(phaseId);
+
+  const { formatMessage } = useIntl();
+
+  const handleProjectFilter = useCallback(({ value }: IOption) => {
+    setProjectId(value);
+    setPhaseId(undefined);
+    setQuestionId(undefined);
+  }, []);
+
+  const handlePhaseFilter = useCallback(({ value }: IOption) => {
+    setPhaseId(value);
+    setQuestionId(undefined);
+  }, []);
+
+  const handleQuestion = useCallback((questionId: string) => {
+    setQuestionId(questionId);
+  }, []);
+
+  const isNativeSurveyPhase =
+    phase?.data.attributes.participation_method === 'native_survey';
+
+  const showAnalyses = isNativeSurveyPhase
+    ? projectId && phaseId && questionId
```

Comment:
```suggestion
    ? projectId && phaseId
```

Maybe it's convenient to just show all analyses of a phase, and make the additional question filter optional?